src/db/postgres/lrsql/postgres/sql/ddl.sql

```diff
@@ -281,6 +280,35 @@ CREATE TYPE scope_enum AS ENUM (
   'profile/read');
 ALTER TABLE IF EXISTS credential_to_scope ALTER COLUMN scope TYPE scope_enum USING (scope::scope_enum);
 
+/* Migration 2024-01-24 - Add profile and profile/read scopes */
+
+/* Simply adding the profile scope will cause a name clash with the reserved
+   OIDC profile scope. As a result, we add prefixes to create activity_profile
+   and agent_profile (and their read-only versions); this has the additional
+   benefit of narrowing the scope to just activity and agent profiles,
+   respectively. Since profile and profile/read have always been unused, we
+   are safe to remove them as enums. */
```

Comment:
Would it be better to simply change the initial migration function (and note the change in the comments) instead of adding an additional migration function (and thus overhead)?

Reply:
Generally it is good practice to keep migrations isolated in to their own code. This becomes a factor if something gets mucked up in production and a database needs to be fixed from the REPL.